l1-contracts/contracts/state-transition/chain-deps/facets/Executor.sol

```diff
@@ -53,7 +56,12 @@ contract ExecutorFacet is ZkSyncHyperchainBase, IExecutor {
         bytes32[] memory blobCommitments = new bytes32[](MAX_NUMBER_OF_BLOBS);
         if (pricingMode == PubdataPricingMode.Validium) {
             // skipping data validation for validium, we just check that the data is empty
-            require(_newBatch.pubdataCommitments.length == 1, "EF: v0l");
+            if (logOutput.pubdataHash != 0x00) {
```

Comment:
This is new error, intended @koloz193 ?

Reply:
let me remove it, i think it was present before we merged in release-v23